### PR TITLE
feat(onboarding): [sc-11513] increase k8swatcher memory limit

### PR DESCRIPTION
**Pull request by nitzangr**
Story details: https://app.shortcut.com/groundcover/story/11513

### CODE REVIEW

File: pkg/helm/presets/backend/high-resources.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ k8sWatcher:
       memory: 300Mi
     limits:
       cpu: 100m
```
Comment (ryechezkel) [inline pkg/helm/presets/backend/high-resources.yaml:8]: also increase the cpu limit to 500mCPU 🙏

**Review by ryechezkel — CHANGES_REQUESTED** [review]

File: pkg/helm/presets/backend/high-resources.yaml (hunk 1 above)
Comment (nitzangr) [reply pkg/helm/presets/backend/high-resources.yaml]: Fixed

File: pkg/helm/presets/backend/high-resources.yaml [hunk 2]
```diff
@@ -4,9 +4,6 @@ k8sWatcher:
     requests:
       cpu: 50m
```
Comment (ryechezkel) [inline pkg/helm/presets/backend/high-resources.yaml:5]: ?

**Review by ryechezkel — CHANGES_REQUESTED** [review]

**Comment by ryechezkel** [comment]
@nitzangr you should be explicit about values as it should override different scenarios

File: pkg/helm/presets/backend/high-resources.yaml (hunk 2 above)
Comment (nitzangr) [reply pkg/helm/presets/backend/high-resources.yaml:5]: @steb-veksler even we do we don't overwrite here, I still think it is better to have the values here explicit

**Review by steb-veksler — APPROVED** [review]

**Review by ryechezkel — APPROVED** [review]

**Pull request merged**